packages/orchestratorlib/src/settings.ts

```
@@ -152,59 +198,57 @@ export class OrchestratorSettings {
     settingsDir = path.resolve(settingsDir);
-    const settingsFile: string = path.join(settingsDir, OrchestratorSettings.OrchestratorSettingsFileName);
-    OrchestratorSettings.SettingsPath = settingsFile;
-    const settingsFileExists: boolean = OrchestratorHelper.exists(settingsFile) || OrchestratorHelper.exists(path.join(settingsDir, 'orchestrator.json'));
-
-    OrchestratorSettings.ModelPath = '';
-    OrchestratorSettings.EntityModelPath = '';
-    OrchestratorSettings.SnapshotPath = '';
+    const settingsFile: string = path.join(settingsDir, settingsFileName);
+    this.SettingsPath = settingsFile;
+    const settingsFileExists: boolean = OrchestratorHelper.exists(settingsFile);
 
     try {
       let settings: any;
       if (settingsFileExists) {
         settings = JSON.parse(OrchestratorSettings.readFile(settingsFile));
-        if (settings.dataSources && settings.dataSources.inputs) {
+        if (settings.dataSources && settings.dataSources.inputs.length > 0) {
           hasDataSources = true;
         }
       }
       if (hasDataSources) {
-        OrchestratorSettings.ensureDataSources(hierarchical, settings, settingsDir);
+        this.ensureDataSources(hierarchical, settings, settingsDir);
       }
-      OrchestratorSettings.SnapshotPath = OrchestratorSettings.ensureSnapshotPath(snapshotPath, settingsDir, settings);
-      OrchestratorSettings.ModelPath = OrchestratorSettings.ensureBaseModelPath(baseModelPath, settings);
+
+      this.SnapshotPath = OrchestratorSettings.ensureSnapshotPath(snapshotPath, settingsDir, settings);
+      this.ModelPath = OrchestratorSettings.ensureBaseModelPath(baseModelPath, settings);
       entityBaseModelPath = OrchestratorSettings.ensureEntityBaseModelPath(entityBaseModelPath, settings);
       if (!Utility.isEmptyString(entityBaseModelPath)) {
-        OrchestratorSettings.EntityModelPath = entityBaseModelPath;
+        this.EntityModelPath = entityBaseModelPath;
       }
-    }  catch {}
+    } catch {}
```

Whats the purpose of an empty catch?